roles/network_validated_content/templates/plugin_directory/plugins/filter/health_check_view.py.j2

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/python
```

Comment:
The general consensus is that we do not want validated content to have Python code. If so, we should not make it a part of best practices.

@cross-logic